dev-requirements.txt

```diff
@@ -5,28 +5,25 @@ git+https://github.com/dbt-labs/dbt-core.git@1.5.latest#egg=dbt-tests-adapter&su
 
 # if version 1.x or greater -> pin to major version
 # if version 0.x -> pin to minor
-black~=23.1
+black~=23.3
 bumpversion~=0.6.0
 click~=8.1
-flake8~=5.0;python_version=="3.7"
 flake8~=6.0;python_version>="3.8"
```

Comment:
I missed this in the PR into `main`, but we don't need the `;python_version>=3.8` on this, `pre-commit`, or 'tox`. I wonder if that's what's causing the `snyk` merge conflict error.